packages/react/src/components/stepper/Stepper.stories.tsx

```diff
@@ -0,0 +1,663 @@
+              return 'available';
+            }
+            if (index === state.activeStep - 1 && activeStepIsValid(state)) {
+              return 'completed';
+            }
+            return stateName;
+          }),
+          fields: {
+            ...state.fields,
+          },
+        };
+      }
+      default:
+        throw new Error();
+    }
+  };
+
+  const initialState = {
+    activeStep: 1,
+    states: ['available', 'disabled', 'disabled', 'disabled'],
+    fields: {
+      firstName: {
+        value: '',
+        visited: false,
+      },
+      lastName: {
+        value: '',
+        visited: false,
+      },
+      age: {
+        value: undefined,
+        visited: false,
+      },
+    },
+  };
+  const labels = ['First name', 'Last name', 'Age', 'Verify and send'];
+
+  const [state, dispatch] = useReducer(reducer, initialState);
+
+  return (
+    <form>
+      <h1 style={{ marginTop: '0', fontSize: '52px', lineHeight: '62px' }}>Simple form example</h1>
+      <Stepper
+        className="stepper-form-validation"
+        labels={labels}
+        language="en"
+        stepHeading
+        states={state.states}
+        selectedStep={state.activeStep}
+        stepsTotal={4}
+        onStepClick={(event, number) => dispatch({ type: 'setActive', payload: number })}
+      />
+
+      <div style={{ height: '250px' }}>
+        {state.activeStep === 1 && (
+          <TextInput
+            style={{ width: '300px', paddingTop: '60px' }}
+            id="firstName"
+            label="First name *"
+            invalid={state.fields.firstName.value.length === 0 && state.fields.firstName.visited === true}
+            errorText={
+              state.fields.firstName.value.length === 0 &&
+              state.fields.firstName.visited === true &&
+              'First name is compulsory'
+            }
+            value={state.fields.firstName.value}
+            onChange={(event) =>
+              dispatch({ type: 'changeField', fieldName: 'firstName', newValue: event.target.value })
+            }
+          />
+        )}
+        {state.activeStep === 2 && (
+          <TextInput
+            style={{ width: '300px', paddingTop: '60px' }}
+            id="lastName"
+            label="Last name *"
+            invalid={state.fields.lastName.value.length === 0 && state.fields.lastName.visited === true}
+            errorText={
+              state.fields.lastName.value.length === 0 &&
+              state.fields.lastName.visited === true &&
+              'Last name is compulsory'
+            }
+            value={state.fields.lastName.value}
+            onChange={(event) => dispatch({ type: 'changeField', fieldName: 'lastName', newValue: event.target.value })}
+          />
+        )}
+        {state.activeStep === 3 && (
+          <NumberInput
+            style={{ width: '300px', paddingTop: '60px' }}
+            id="age"
+            label="Age *"
```

Comment:
Rather than adding an asterisk to the label, should we use the `required` property?

Reply:
Yes:

- [ ] Add missing required prop